package.json

```diff
@@ -35,4 +35,4 @@
     "jest": "^26.4.2",
     "standard": "^16.0.4"
   }
-}
+}
```

Comment:
Why did you remove the newline?

Reply:
my editor by default removes that line on save, wait just a second.